iOS/FVM/Framework/Controllers/FVMDamagedCarViewController.swift

```diff
@@ -81,25 +80,21 @@ public class FVMDamagedCarViewController: UIViewController {
     }
     
     @objc
-    fileprivate func disableAcceptButton() {
+    private func disableAcceptButton() {
         acceptButton.setTitleColor(UIColor.lightGray, for: .disabled)
         acceptButton.alpha = 0.5
         acceptButton.isEnabled = false
     }
     
     @objc
-    fileprivate func enableAcceptButton() {
+    private func enableAcceptButton() {
         acceptButton.setTitleColor(UIColor.blue, for: .normal)
         acceptButton.alpha = 1.0
         acceptButton.isEnabled = true
     }
     
     @objc
-    fileprivate func hideRotationPrompt() {
-        for view in self.view.subviews {
-            if view.restorationIdentifier?.isEqual(ROTATION_PROMPT) ?? false as Bool {
```

Comment:
Const field ROTATION_PROMPT is no longer used.
Please remove its declaration from code (FVMDamagedCarViewController -> line 26)